pkg/client/cache/listwatch_test.go

```diff
@@ -117,7 +117,7 @@ func TestListWatchesCanWatch(t *testing.T) {
 		{
 			location: buildLocation(
 				testapi.Default.ResourcePathWithPrefix("watch", "nodes", api.NamespaceAll, ""),
-				buildQueryValues(url.Values{"resourceVersion": []string{""}})),
+				buildQueryValues(url.Values{})),
```

Comment:
To clarify more - I think we need a consistent behavior between what is produced by:
<perResourceClient>.Watch(options)
and
<listWatcher>.Watch(options)
And this change is making this consistent.